CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ SET_PROPERTY (TARGET binlogevents PROPERTY IMPORTED_LOCATION ${MYSQL_BIN}/libbin
 
 ADD_DEFINITIONS (-DDBUG_OFF)
 SET (LINK_TYPE STATIC)
-SET (MYSQL_LIBS mysqlclient binlogevents -lssl -lcrypto)
+SET (MYSQL_LIBS mysqlclient binlogevents -lssl -lcrypto -ldl)
```

Comment:
It is rule of thumb to use [CMAKE_DL_LIBS](https://cmake.org/cmake/help/latest/variable/CMAKE_DL_LIBS.html) instead of `-ldl`. Different OSes may require it or not: say, FreeBSD didn't have libdl.so until 12th version.